modules/RTC/RTCUtils.js

```diff
@@ -753,10 +759,12 @@ var RTCUtils = {
                 });
             } else {
                 try {
-                    logger.error('Browser does not appear to be WebRTC-capable');
+                    logger.error(
+                        'Browser does not appear to be WebRTC-capable');
                 } catch (e) {
                 }
```

Comment:
@lyubomir I haven't added the try catch in this commit. Maybe I have added it a long time ago in the code but I don't remember why. I think logger.error shouldn't throw anything and we should remove the try catch but I don't think it's related to the current PR. WDYT?